project_6.html

```diff
@@ -0,0 +1,985 @@
+<!DOCTYPE html>
```

Comment:
Overall, the entire page is *not* mobile responsive. See the other projects and the newly resurrected template for what the mobile version should be.

Template: 
![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/75911565/44c4d762-93fe-4db3-bd3e-fb24289f70d1)


Your version:
![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/75911565/9c754c16-08d6-4721-865c-e99eb3181483)

Specifically, you can scroll left and right. It seems like the text width isn't being limited.